### DISCUSSION TITLE
Last Fm on Navidrome / Windows

### DISCUSSION THREAD

**Discussion by coacharnold1** [navidrome/navidrome#2874] [Q&A]
HI FOlks ...  just playing around with navidrome on a windows laptop,  I normally run it on fedora ...       I don't  see LastFM in the web gui for windows and I have enabled it in my .toml file ....   I get listenbrainz by default and no last fm in the gui....

is this even working in the windows fork?

this is what I have in the config .toml

LISTENBRAINZ_ENABLED = false
LASTFM_ENABLED = true

#my secret stuff from my linux navidrome.yml
LASTFM_APIKEY = "xxxxxxxxxx"
LASTFM_SECRET = "xxxxxxxxxx"

the only reason I'm doing this is as a proof of concept to show some windows based friends.

thanks for any help

Tim 

![image](https://github.com/navidrome/navidrome/assets/30500110/8fe317f1-fb6d-4b47-9f6d-857b967e9016)

**Comment by metalheim** [comment]
Your configuration values are wrong.
Please take the exact configurations as described in the documentation https://www.navidrome.org/docs/usage/configuration-options/#advanced-configuration

For example, it is "LastFM.Enabled" not LASTFM_ENABLED

**Reply by coacharnold1** [reply]
thanx ...  confusing going back and forth between the .toml and the ,yml
